templates/tableau-cluster-linux.template

```diff
@@ -0,0 +1,1627 @@
+            "MaxLength":"30"
+        },
+        "Password":{
+            "Description":"Tableau Services Manager (TSM) administrator password",
+            "Type":"String",
+            "MinLength":"6",
+            "NoEcho":"true"
+        },
+        "TableauServerAdminUser":{
+            "Description":"The name of the initial administrator for Tableau Server",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "TableauServerAdminPassword":{
+            "Description":"The password of the initial administrator for Tableau Server",
+            "Type":"String",
+            "MinLength":"1",
+            "NoEcho":"true"
+        },
+        "TableauServerLicenseKey":{
+            "Description":"License Key",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegFirstName":{
+            "Description":"First Name",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegLastName":{
+            "Description":"Last Name",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegEmail":{
+            "Description":"Email",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegCompany":{
+            "Description":"Company",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegTitle":{
+            "Description":"Title",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegDepartment":{
+            "Description":"Department",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegIndustry":{
+            "Description":"Industry",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegPhone":{
+            "Description":"Phone",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegCity":{
+            "Description":"City",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegState":{
+            "Description":"State",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegZip":{
+            "Description":"ZIP/Postal Code",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "RegCountry":{
+            "Description":"Country",
+            "Type":"String",
+            "MinLength":"1"
+        },
+        "VpcId":{
+            "Description":"The ID of the VPC into which to deploy the cluster",
+            "Type":"AWS::EC2::VPC::Id"
+        },
+        "ExistingSecurityGroup":{
+            "Description":"Provide an existing security group or leave blank for creating a new one",
+            "Type":"String"
+        },
+        "IPAddress":{
+            "Description":"Provide an Elastic IP address (EIP) for the initial node (blank = no EIP)",
+            "Type":"String"
+        }
+    },
+    "Mappings":{
+        "AwsRegionToAMI":{
```

Comment:
This map must be called AWSAMIRegionMap as specified here: https://aws-quickstart.github.io/use-amis.html